apps/homescreen/js/app.js

```diff
@@ -436,7 +443,7 @@ const SETTINGS_VERSION = 0;
       this.handleEvent({ type: 'scroll' });
     },
 
-    snapScrollPosition: function(bias) {
+    refreshMetrics: function(snap, bias) {
```

Comment:
I'm not really a fan of this function naming, afterall, naming things is the hardest problem in computer science. When I think metrics, I think statistics collected by some server.

My preference here would be an individual getter for both snap and bias, more clarity that way and avoids the boolean trap you have now.